Documentation/Configuration.rst

```diff
@@ -35,21 +35,29 @@ Where `<Physical interface>` is the name of the interface that the bridge should
 
 * To create a network on the node, perform the following as a superuser::
 
-    sudo mcvirt network create <Network name> --interface <Bridge interface>
+    mcvirt network create <Network name> --interface <Bridge interface>
 
 
 * Assuming that there are not any VMs connected to a network, they can be removed using::
 
-    sudo mcvirt network delete <Network name>
+    mcvirt network delete <Network name>
 
 Configure MCVirt
 -----------------
 
 * The first time MCVirt is run, it creates a configuration file for itself, found in **/var/lib/mcvirt/<Hostname>/config.json**.
 * The volume group, in which VM data will be stored as logical volumes, must be setup using::
 
-    sudo mcvirt node --set-vm-vg <Volume Group>
+    mcvirt node --set-vm-vg <Volume Group>
 
 * The cluster IP address must be configured if the node will be used in a cluster (See the `Cluster documentation <Cluster.rst>`_)::
 
-    sudo mcvirt node --set-ip-address <Cluster IP Address>
+    mcvirt node --set-ip-address <Cluster IP Address>
+
+* If the node will be used in a cluster, the hosts file at ``/etc/hosts`` must edited by changing the line::
```

Comment:
This needs to be done even if the node isn't added to a cluster, as it is required for the RPC client to connect to the local machine